src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -78,28 +101,23 @@ def test_single_property_setting():
         "CACHE_DIR": "./",
         properties.inference_num_threads(): 9,
         properties.affinity(): "NONE",
-        "INFERENCE_PRECISION_HINT": Type.f32,
-        properties.hint.performance_mode(): properties.hint.PerformanceMode.LATENCY,
-        properties.hint.num_requests(): 12,
         "NUM_STREAMS": properties.streams.Num(5),
     },
 ])
 def test_properties_core(properties_to_set):
```

Comment:
Can you add just properties tests separetely? Parametrized test for every entry. Let me know if I can take care of it.

Reply:
discussed offline: will be fixed by separate PR